### PR TITLE
Fitnesse config update

### PR DESCRIPTION
**Pull request by luwangshell**

### CODE REVIEW

File: plugins.properties [hunk 1]
```diff
@@ -1,12 +1,10 @@
 Port=8080
-SLIM_PORT=8475
 TEST_SYSTEM=slim
 slim.timeout=10
-FITSHARP_HOME=C:\\Apps\\FitNesse\\FitSharp
+FITSHARP_HOME=C:\\Apps\\FitNesse\\FitSharp\\lib\\net5.0
```
Comment (essenius) [inline plugins.properties]:
```suggestion
FITSHARP_HOME=C:\\Apps\\FitSharp\\2.8.2.1\\net5.0
```

File: FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki [hunk 2]
```diff
@@ -1,3 +1,3 @@
-!path "Rest\Rest\bin\Release\Rest.dll;Rest\RestTests\bin\release\RestTests.dll"
-!define RestTestDll {Rest\RestTests\bin\Release\RestTests.dll}
+!path "Rest.dll;RestTests.dll"
```
Comment (essenius) [inline FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki]: remove. Also remove the import table in SuiteSetup and LoadSaveTest

File: FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
+!define RestTestDll {RestTests.dll}
```
Comment (essenius) [inline FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki:1]: remove

File: README.md [hunk 4]
```diff
@@ -2,19 +2,25 @@
...
-3. Clone the repo to a local folder (C:\Data\FitNesseDemo)
-4. Update plugins.properties to point to the FitSharp folder (if you took other folders than suggested)
-5. Build all projects in the solution Rest (Release)
-6. Ensure you have Java installed (1.7 or higher)
-7. Start FitNesse with the root repo folder as the data folder as well as the current directory:
+1. Download FitNesse (http://fitnesse.org) and install it to <I>C:\Apps\FitNesse</I>
+2. Download FitSharp (https://github.com/jediwhale/fitsharp) and install it to <I>C:\Apps\FitNesse\FitSharp</I>.
+   <br />Or install the package via nuget: 
+   ```
+   nuget install FitSharp
```
Comment (essenius) [inline README.md]: If you do it via NuGet, you either need to copy the net5.0 DLLs to C:\Apps\FitSharp\2.8.2.1\net5.0 or change the FITSHARP_HOME variable in plugins.properties to where NuGet puts those DLLs ${USERPROFILE}\\.nuget\\packages\\fitsharp\\2.8.2.1\\lib\\net5.0

File: Rest/Rest/config.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<suiteConfig>
+  <ApplicationUnderTest>
+    <AddAssembly>Rest.dll</AddAssembly>
+    <AddNamespace>Rest</AddNamespace>
```
Comment (essenius) [inline Rest/Rest/config.xml:5]: Add namespace rest.ContentHandlers

File: plugins.properties [hunk 6]
```diff
@@ -1,12 +1,10 @@
...
-TEST_RUNNER=${FITSHARP_HOME}\\Runner.exe
+FIXTURE_FOLDER=.
+COMMAND_PATTERN=%m ${FITSHARP_HOME}\\Runner.dll -c ${FIXTURE_FOLDER}\\config.xml
+TEST_RUNNER=dotnet
 VersionsController.days=0
-SeleniumServer=!-http://localhost:6667-!
-TESTPAGE=!-http://seleniumfixturedemo.azurewebsites.net//TestPage.html-!
-BROWSER=Chrome Headless
\ No newline at end of file
+BROWSER=Chrome Headless
```
Comment (essenius) [inline plugins.properties]: remove

**Review by essenius — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (luwangshell) [reply README.md]: in Nuget install, you can specify the output folder with switch: -OutputDirectory (https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-install). e.g. nuget install FitSharp -OutputDirectory C:\Apps\FitNesse

File: FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki (hunk 2 above)
Comment (luwangshell) [reply FitNesseRoot/FitSharpDemos/RestSuite/_root.wiki]: Removed.

File: Rest/Rest/config.xml (hunk 5 above)
Comment (luwangshell) [reply Rest/Rest/config.xml:5]: ContentHandlers object is also under namespace _**Rest**_.  Verified that no need to add to config.xml file.

**Comment by luwangshell** [comment]
@essenius I have updated the changes and also included the setup for Mac. Could you pls review?

File: plugins.properties [hunk 7]
```diff
@@ -1,9 +1,20 @@
...
-FITNESSE_ROOT=${FITNESSE_ROOTPATH}\\FitNesseRoot
+FITSHARP_HOME=C:\\Apps\\FitSharp\\2.8.2.1\\net5.0 
+FITNESSE_ROOT=${FITNESSE_ROOTPATH}\\FitNesseRoot 
 FIXTURE_FOLDER=.
-COMMAND_PATTERN=%m ${FITSHARP_HOME}\\Runner.dll -c ${FIXTURE_FOLDER}\\config.xml
+COMMAND_PATTERN=%m ${FITSHARP_HOME}\\Runner.dll -c ${FIXTURE_FOLDER}\\config.xml 
 TEST_RUNNER=dotnet
 VersionsController.days=0
+
+! Setup for Mac !
```
Comment (essenius) [inline plugins.properties]: Comments start with a # in this config format

**Review by essenius — APPROVED** [review]

**Pull request merged**